tests/BIAnalyticsManagementServiceTests/GetParticipantScreeningEpisodeTests/GetParticipantScreeningEpisodeTests.cs

```diff
@@ -59,13 +59,13 @@ public async Task Run_Should_Return_OK_With_EpisodesDataPage_Json_Data()
         {
             { "page", "1" },
             { "pageSize", "2" },
-            { "startDate", "2023-07-05 08:30:00" },
-            { "endDate", "2023-07-05 08:30:00" }
+            { "startDate", "2023-05-05 08:30:00" },
+            { "endDate", "2023-05-05 08:30:00" }
```

Comment:
I think using the date 2023-07-05 is a better test since it would catch any potential issue with the month and day being flipped. Could you make this change in all the testing using 2023-07-05, please